recipe/meta.yaml

```diff
@@ -15,15 +15,27 @@ build:
 
 requirements:
     build:
+        - toolchain
         - gcc
         - perl
+        - gcc  # [osx]
+        - libgfortran  # [linux]
     run:
-        - gcc
+        - libgfortran  # [not win]
```

Comment:
Unfortunately due to... Reasons... conda-forge decided to package `libgfortran` with `libquadmath` on `OS X` but on `Linux` it is only `libgfortran`. Due to this inconsistency if you need `libquadmath` on Linux you will need to do:

```yaml
    - libgfortran  # [osx]
    - libgccc  # [linux]
```

I am still fighting to get a modern `gcc` package on `conda-forge` and stop this madness. Any help from users is much appreciated.